src/tests/src/solver/modeler/api/CMakeLists.txt

```diff
@@ -1,20 +1,2 @@
-set(EXECUTABLE_NAME tests-modeler-api-ortools)
-add_executable(${EXECUTABLE_NAME} testApiOrtoolsLinearProblem.cpp)
-
-target_include_directories(${EXECUTABLE_NAME}
-        PRIVATE
-        "${src_solver_optimisation}"
-)
-
-target_link_libraries(${EXECUTABLE_NAME}
-        PRIVATE
-        Boost::unit_test_framework
-        Antares::modeler-ortools-impl
-)
-
-# Storing tests-ts-numbers under the folder Unit-tests in the IDE
-set_target_properties(${EXECUTABLE_NAME} PROPERTIES FOLDER Unit-tests)
-
-add_test(NAME test-modeler-api-ortools COMMAND ${EXECUTABLE_NAME})
-
-set_property(TEST test-modeler-api-ortools PROPERTY LABELS unit)
+include(LinearProblemWithOrtools.cmake)
+include(LinearProblemBuilder.cmake)
```

Comment:
this will produce two executable, one for each .cmake, i think it's not necessary;
having more exe to build as we add test is too costly, consider 1 exe for multiple unit translations

Reply:
When you say "too costly", what do you mean ?
And if you're right, which unit tests policy should we apply ?
For example, should we necessarily regroup unit tests of a sub directory in an executable ?
At which level of sub directory do we stop gathering unit tests ?

We should discuss about this, as a team, and identify what we want to do exactly : Is it really suitable ? Are there any drawback or benefit ?

Reply:
generating 2 exe take more times and memory than one exe, right?
multiply this for every lib we would create:
lib_test
 |_ sublib1_exe
 |_ sublib2_exe                      VS     lib_test_exe
 : 
 |_ sublibn_exe